cl/api/utils.py

```diff
@@ -363,6 +363,12 @@ class RECAPUploaders(DjangoModelPermissions):
     }
 
 
+class EmailProcessingQueueAPIUsers(BasePermission):
+    def has_permission(self, request, view):
+        # TODO - This will check against an API token in the header. This will be stored in environment variables on either side.
```

Comment:
We will add this after the key strategy is determined on the AWS Cloud side.

Reply:
If you want to get this merged finally, I'm happy to keep this as is. Let me know. If so, I'll make an task somewhere so we remember to come back around to it.

Reply:
Got it #1676 

Added that to the project board as well.

Reply:
Also adding a link to the issue to the comment itself.